iteration.go

```diff
@@ -233,6 +262,39 @@ func createIterationLinks(request *goa.RequestData, id interface{}) *app.Generic
 	}
 }
 
+// ConvertFromLtreeFormat converts data to UUID format from ltree format.
+func ConvertFromLtreeFormat(uuid string) string {
```

Comment:
yes, thanks, Done in https://github.com/almighty/almighty-core/pull/713/commits/496a66aefb24e66e665eb72d3b014098c7d76eb0